### ISSUE TITLE
Unable to login to yax.im XMPP from Firefox OS 1.3

### ISSUE THREAD

**Issue by ossguy** [loqui/im#649]
When I try to login to my @yax.im XMPP account on my ZTE Open C running Firefox OS 1.3 using the Loqui IM app installed from the Marketplace on 2015-02-24, I see it "Connecting" for a few seconds, then "Synchronizing" for a few seconds, then it returns to the login screen and does not indicate any type of error.  If I click "Log in" again, I get the same result.

I also tried logging in to another XMPP account I have (on a private server) with the same result.  I can login to both accounts fine using Gajim on my desktop.

**Comment by TitanNano** [comment]
Do these accounts have a vCard? Loqui is unable to setup accounts without a vCard.

**Comment by ossguy** [comment]
I confirmed that the login succeeded when a vCard was added.  Still, it seems that I should have at least been presented with an error message indicating the login failed (better yet, a message that told me why).  As it is, it's very unclear if a login was even attempted, or if Loqui is buggy, or if there's some other unrelated issue affecting the login, since no error messages is printed when it fails to sign in due to lack of vCard.
